backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/cardlist/BpkCardList.kt

```diff
@@ -0,0 +1,57 @@
+/**
+ * Backpack for Android - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.skyscanner.backpack.compose.cardlist
+
+import androidx.compose.foundation.lazy.LazyItemScope
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import net.skyscanner.backpack.compose.cardlist.internal.BpkCardListImpl
+
+@Composable
+fun <T> BpkCardList(
+    title: String,
+    description: String,
+    layout: BpkCardListLayout,
+    dataList: List<T>,
```

Comment:
rather than providing this (the component doesn't need to know anything about the data) i'd follow what we do in `BpkCarousel` and use a state here. we can probably reuse the same state (we do that in a few other components)

Reply:
great point. Just removed and added the minimum cards to show instead - which is handy because we need it for the "adapter"